### PR TITLE
feature/pinterest-keywords-variable-addition

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** #77 

**This PR will result in the following new package version:** `v1.4.0`

While this isn't technically breaking, it will be included in the next major release. As such we will mark it as breaking.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

This PR includes a new variable `pinterest__using_keywords` that allows users to disable the keywords reports within Pinterest if they are not needed or are not located within the users source schema.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt compile
- [X] dbt run –full-refresh
- [X] dbt run
- [X] dbt test
- [X] dbt run –vars (pinterest__using_keywords set to both true and false)

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked and tagged
- [X] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [X] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [X] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [X] You have provided details below around the validation steps performed to gain confidence in these changes.

- Variable set to `false` with all other ad platforms (excluding amazon ads) enabled.
  - The keyword tables are not included 
image

- Variable set to `true` with all other ad platforms (excluding amazon ads) enabled.
  - The keyword tables are included
image

- Variable not defined (default) with all other ad platforms (excluding amazon ads) enabled.
  - The keyword tables are included
image

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [X] indexed within dbt_project.yml
    - [X] indexed within integration_tests/dbt_project.yml
- [X] CHANGELOG has individual entries for each respective change in this PR
    
- [X] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [n/a] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [X] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
📚

### CODE REVIEW

**Review by fivetran-reneeli — CHANGES_REQUESTED** [review]
Changes look good but don't forget to update the version!

**Comment by fivetran-joemarkiewicz** [comment]
> Changes look good but don't forget to update the version!

Thanks for catching @fivetran-reneeli, this is going to be merged into the release/v1.4.0 branch so the version should be bumped when merging into there. If all else looks good would you be able to approve or add any other comments before merging to the release branch. Thanks!

**Comment by fivetran-reneeli** [comment]
Looks good!

**Review by fivetran-reneeli — APPROVED** [review]
Lgtm!

File: packages.yml [hunk 1]
```diff
@@ -11,8 +11,12 @@ packages:
...
     version: [">=0.9.0", "<0.10.0"]
 
-  - package: fivetran/pinterest
-    version: [">=0.7.0", "<0.8.0"]
+  # - package: fivetran/pinterest
+  #   version: [">=0.7.0", "<0.8.0"]
+
+  - git: https://github.com/fivetran/dbt_pinterest.git
+    revision: feature/keywords-variable-addition
+    warn-unpinned: false
```
Comment (fivetran-catfritz) [inline packages.yml:20]: Standard reminder.

**Pull request merged**